conformance/client/google-java/src/main/kotlin/com/connectrpc/conformance/client/java/Main.kt

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022-2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.connectrpc.conformance.client.java
+
+import com.connectrpc.conformance.client.Client
+import com.connectrpc.conformance.client.ConformanceClientLoop
+
+fun main(args: Array<String>) {
+    try {
+        val invokeStyle = ConformanceClientLoop.parseArgs(args)
+        val client = Client(
+            invokerFactory = ::JavaInvoker,
+            serializationFactory = JavaInvoker::serializationStrategy,
+            invokeStyle = invokeStyle,
+        )
+        ConformanceClientLoop.run(System.`in`, System.out, client)
+    } catch (e: Exception) {
+        e.printStackTrace(System.err)
+        Runtime.getRuntime().exit(1)
```

Comment:
If we want we could just let exceptioon bubble up which should print the exception and exit with a non-zero exit code on its own.